src/cf-typography/src/molecules/list.less

```diff
@@ -0,0 +1,103 @@
+//
+// Unstyled list modifier
+//
+
+.m-list__unstyled,
+.m-list__horizontal,
+.m-list__icons,
+.m-list__links {
+    padding-left: 0;
+
+    list-style-type: none;
+
+    // This is needed for dd elements.
+    .a-list-item {
+        margin-left: 0;
+    }
+}
+
+//
+// Spaced list modifier
+//
+
+.m-list__spaced {
+    .m-list__spaced,
+    .a-list-item + .a-list-item {
+        margin-top: unit( 24px / @base-font-size-px, em );
+    }
+}
+
+//
+// Horizontal list modifier
+//
+
+.m-list__horizontal {
+    .a-list-item {
+        display: inline-block;
+
+        // Assuming a natural space of 4px between inline block items
+        // then the space between would be 8px (4px natural + 4px added).
+        margin-right: unit( 4px / @base-font-size-px, em );
+    }
+}
+
+//
+// Icon list modifier
+//
+
+.m-list__icons {
+    .a-list-item {
+        padding-left: 1.5em;
+        position: relative;
+    }
+
+    .cf-icon {
+        position: absolute;
+        left: 0;
+        width: 1.5em;
+        text-align: center;
+        top: 0.1em;
+    }
+}
+
+//
+// Link list modifier
+//
+
+.m-list__links {
+    .a-list-item {
+        .respond-to-max( @bp-xs-max, {
+            margin-bottom: 0;
+
+            &:nth-child(n+2) .a-link {
+                border-top-width: 0;
+            }
+        } );
+    }
+
+    .a-link {
```

Comment:
I'm not following. Do you mean `.a-link.m-list_link`?

This is an awfully theoretical discussion, since `.a-link` has no styles and probably should never :)